core/pkg/ingress/annotations/ipwhitelist/main.go

```diff
@@ -56,8 +56,9 @@ func (a ipwhitelist) Parse(ing *extensions.Ingress) (interface{}, error) {
 	sort.Strings(defBackend.WhitelistSourceRange)
 
 	val, err := parser.GetStringAnnotation(whitelist, ing)
-	if err != nil {
-		return &SourceRange{CIDR: defBackend.WhitelistSourceRange}, err
+	// A missing annotation is not a problem, just use the default
+	if err == ing_errors.ErrMissingAnnotations {
+		return &SourceRange{CIDR: defBackend.WhitelistSourceRange}, nil
```

Comment:
`&SourceRange{CIDR: defBackend.WhitelistSourceRange}, err` is what it used to return, and the value was ignored because of the error, so I don't _think_ so. But I'm not sure what's causing the issue.